src/custom-elements/src/components/dropdown/dropdown.tsx

```diff
@@ -35,6 +36,10 @@ export class Dropdown {
    * To provide the value of base-style button as trigger of the Dropdown
    */
   @Prop() button?: string;
+  /**
+   * To animate the accordion of Dropdown
+   */
+  @Prop() animate?: boolean;
```

Comment:
@soujanreddy this prop is making the PR failing when build. One option would be to change the name to `iconAnimation` or something along these lines